milvus/ClientProxy.go

```diff
@@ -0,0 +1,224 @@
+package milvus
+
+import (
+	"flag"
+	pb "github.com/milvus-io/milvus-sdk-go/milvus/grpc/gen"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/testdata"
+	"log"
+)
+
+var (
+	tls                = flag.Bool("tls", false, "Connection uses TLS if true, else plain TCP")
+	caFile             = flag.String("ca_file", "", "The file containing the CA root cert file")
+	serverHostOverride = flag.String("server_host_override", "x.test.youtube.com", "The server name use to verify the hostname returned by TLS handshake")
```

Comment:
why youtube.com?

Reply:
I put it here just as a template. I will delete it later